scripts/app-stream/buildImage.ps1

```diff
@@ -15,6 +18,11 @@ cd "C:\Program Files\Amazon\Photon\ConsoleImageBuilder"
 
 .\image-assistant.exe add-application --absolute-app-path "C:\Program Files (x86)\Mozilla Firefox\firefox.exe" --display-name Firefox --name Firefox
 
+.\image-assistant.exe add-application --absolute-app-path "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" `
+--display-name MicrosoftRemoteDesktop --name MicrosoftRemoteDesktop `
+--launch-parameters '`"-File "C:\Users\Public\Documents\MicrosoftRemoteDesktop.ps1" -ExecutionPolicy Bypass`"' `
+--working-directory 'C:\Users\public\Documents\'
```

Comment:
Why is `working-directory` needed here?

Reply:
Just added a directory that is common between images and image builder, and I found that mstsc functions appropriately with that working directory.